pkg/scheduler/core/generic_scheduler.go

```diff
@@ -231,6 +223,14 @@ func (g *genericScheduler) Schedule(ctx context.Context, state *framework.CycleS
 			FilteredNodesStatuses: filteredNodesStatuses,
 		}
 	}
+	trace.Step("Computing predicates done")
+
+	// Run "postfilter" plugins.
+	postfilterStatus := g.framework.RunPostFilterPlugins(ctx, state, pod, filteredNodes, filteredNodesStatuses)
```

Comment:
If we move post filter plugin here, the `PostFilter` won't be called even if they have to do some cleanup.